aircraft/breguet_range.py

```diff
@@ -0,0 +1,53 @@
+"""Breguet range model"""
+from gpkit.shortcuts import Var, Model
+from gpkit.tools import te_exp_minus1
+
+class BreguetRange(Model):
+
+    """Breguet Range Model
+
+    Assumptions
+    -----------
+    Fuel burn varies linearily with thrust and is independent of velocity.
```

Comment:
You're assuming you're looking at the docstring someplace other than in the code, such that scrolling down to see the units defined in the `Var` calls is not possible?